### DISCUSSION TITLE
Refiner link color on dark background issue

### DISCUSSION THREAD

**Discussion by LetMeSP** [microsoft-search/pnp-modern-search#1997] [Q&A] [answered]
After updating to the newest version refiner filter links are still dark when changing to the dark background. Before it was white. Is this a bug or am I missing something? Screenshot for reference.

![2022-04-04_11-02-45](https://user-images.githubusercontent.com/41652178/161605196-59e5eaf5-1c21-4f5a-8c2d-2269195c87a9.png)

**Comment by FranckyC** [comment, marked as answer]
Fixed with #2133
